### PR TITLE
[slackana-11] - [Markup] Create Settings Modal in Profile Avatar Dropdown

### PR DESCRIPTION
**Pull request by johnpaul-sun**
## Issue Link
-  https://app.asana.com/0/1203011167276287/1203026696604056/f

## Definition of Done
- [x] User can open settings modal
- [x] User can select in the settings menu Profile, Security, Notification
- [x] User can toggle the toggle switch button in the notification

## Notes 
- (Optional)

## Pre-condition
- yarn

## Expected Output
- The settings modal should popup after the user clicked the profile button
- The settings modal should close if the user clicked the close button and clicked outside the modal

## Screenshots/Recordings
- Profile
image

- Security
image
 
- Notification
image

### CODE REVIEW

File: client/src/components/templates/DialogBox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+              </Dialog.Panel>
+            </Transition.Child>
+          </div>
+        </div>
+      </Dialog>
+    </Transition>
+  )
+}
+
+export default DialogBox;
```
Comment (abduljalilpalala) [inline client/src/components/templates/DialogBox/index.tsx]: Add new line

File: client/src/shared/interfaces/index.ts [hunk 2]
```diff
@@ -11,3 +11,13 @@ export interface Link {
...
+<<<<<<< HEAD
+
+export interface ProjectList {
+  team_name: string,
+  icon: string,
+  date: string,
+  status: string,
+}
+=======
+>>>>>>> develop
```
Comment (abduljalilpalala) [inline client/src/shared/interfaces/index.ts]: Remove `<<<<<<< HEAD` and `>>>>>>> develop`

File: client/src/shared/twin/home-content.style.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  body: [tw`px-6 py-3 rounded-t-lg border border-[${slate_300}]`],
+  bodyHeader: [tw`flex flex-row justify-between content-between`],
+  addProjectDiv: [tw`flex flex-row items-center justify-center gap-3`],
+  addProjectButton: [tw`hover:bg-[${slate_100}] cursor-pointer w-12 h-12 flex items-center justify-center rounded-lg border-dotted border border-[${slate_400}]`],
+  dropDown: [tw`hover:bg-[${slate_100}] cursor-pointer w-12 h-12 flex items-center justify-center rounded-lg border border-[${slate_500}]`],
+  container: [tw`z-10 border-b rounded-b-lg border-[${slate_300}] mobile:grid-cols-1 tablet:grid-cols-2 px-7 py-10 grid grid-cols-3 gap-6 h-px-400 overflow-y-scroll`],
+  more: [tw`text-[${slate_600}] hover:text-[${slate_400}] ease-in-out duration-300 font-medium cursor-pointer flex justify-start items-center`],
+  tags: [tw`flex flex-col justify-start items-start`],
+  dropDownMenu: [tw`absolute z-20 right-0 mt-2 w-56 origin-top-right divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none`]
+}
```
Comment (abduljalilpalala) [inline client/src/shared/twin/home-content.style.ts]: Add new line

File: client/src/shared/twin/settings-modal.style.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  upload: [tw`w-px-170 h-px-36 tracking-tight bg-blue-600 text-[${slate_50}] rounded-md hover:bg-blue-800 ease-in-out duration-150`],
+  remove: [tw`w-px-170 border h-px-36 tracking-tight border-blue-600 text-[${slate_900}] rounded-md hover:bg-[${slate_500}] hover:text-[${slate_50}] ease-in-out duration-150`],
+  uploadContainer: [tw`flex flex-row justify-between items-center`],
+  active: [tw`border-b border-[${slate_900}] text-[${slate_900}]`],
+  inactive: [tw`text-[${slate_400}] outline-none`],
+  notification: [tw`flex flex-row justify-between items-start border-b border-[${slate_300}] pb-5`],
+  p: [tw`text-[${slate_900}] text-[15px] text-left mb-5`],
+  disabled: [tw`flex flex-row justify-between items-start pt-3 opacity-50`],
+  value: [tw`text-[${slate_900}] text-[15px] text-left`],
+}
```
Comment (abduljalilpalala) [inline client/src/shared/twin/settings-modal.style.ts]: Add new line

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]

**Review by jsvelte — COMMENTED** [review]
Approved 💯

**Review by abduljalilpalala — APPROVED** [review]

**Review by ipilikent — COMMENTED** [review]
## Approved PR
-  All displayed is the same as the Figma design
-  The Setting Modal is responsive
-  Toggle the toggle switch button is function

## Proof
<details>

Setting Modal


</details>

Test Case : https://docs.google.com/spreadsheets/d/1pa_tZVA7ar1JsgKeIS_0JTaJX9ZywKrX/edit#gid=1229389153
-----
- Total Test time: 1.6 hrs
- Total Comments: 0
- Total Iterations: 1

**Pull request merged**